.github/ISSUE_TEMPLATE/work-item-issue.md

```diff
@@ -0,0 +1,81 @@
+# Work Item Issue Template
```

Comment:
Which of the below items are mandatory vs optional? I know you mention optionality at the bottom, but it's still not clear to me.

Reply:
I have included an asterisk (*) at the end of the title, see below:
![image](https://github.com/user-attachments/assets/c901ca59-7e5f-4e5b-8daa-d7ee2d176227)